CHANGELOG.md

```diff
@@ -19,9 +19,7 @@ Release Versions
 ## Upcoming changes
 
 - fix(robot-model): improve ik performance (#205)
-- fix(robot-model): kinematics tests (#207)
 - feat: add integrate and differentiate methods to Cartesian and joint types (#193)
```

Comment:
The CI wants me to change the changelog, so I decided to clean it up a bit, it is issue 205 that captures the changes for the IK so we don't need 3 lines for it